weblogic-azure-aks/src/main/arm/scripts/setupWLSDomain.sh

```diff
@@ -383,13 +383,13 @@ function query_acr_credentials() {
     # to mitigate error in https://learn.microsoft.com/en-us/answers/questions/1188413/the-resource-with-name-name-and-type-microsoft-con
```

Comment:
Fix the ACR not found issue by registering resource type and specifying resource group name.